apps/labs/posts/captioning-a-newcomers-guide.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Captioning: A Newcomer’s Guide'
+published: February 1, 2024
```

Comment:
+/- a day won't matter much. By the time this goes live it'll be Wednesday somewhere on Earth, anyways, so this'll be accurate either day it goes live.

```suggestion
published: January 24, 2024
```